common/containers/OnboardModal/components/BlockchainSlide.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import onboardIconFour from 'assets/images/onboarding_icon-04.svg';
+
+interface Props {
+  setOnboardStatus(slideNumber: number): void;
+}
+
+const BlockchainSlide: React.SFC<Props> = ({ setOnboardStatus }) => {
+  return (
+    <article className="onboarding__modal">
```

Comment:
Would prefer sticking with our [existing style nomenclature](https://github.com/MyEtherWallet/MyEtherWallet#style-guides-and-philosophies).

Reply:
I didn't do any styles just in case there was a mockup I was missing but I will use the style nomenclature

Reply:
OK cool, then you could also ditch these classes if they're doing nothing rather than changing them all.